actionview/lib/action_view/caching_registry.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module ActionView
+  # = Action View Caching registry
+  class CachingRegistry #:nodoc:
+    extend ActiveSupport::PerThreadRegistry
+
+    attr_accessor :is_caching
```

Comment:
Let's add the extra line to get the predicate method:

```suggestion
    attr_accessor :caching
    alias caching? caching
```